.github/workflows/build.yml

```diff
@@ -69,6 +69,38 @@ jobs:
           path: |
             build/coreboot.rom
           retention-days: 30
+  build_optiplex:
+    runs-on: ubuntu-22.04
+    container:
+      image: coreboot/coreboot-sdk:2023-11-24_2731fa619b
+      options: --user 1001
+    strategy:
+      matrix:
+        vendor: [ dell ]
+        model: [ optiplex_9010_sff_uefi_txt ]
```

Comment:
```suggestion
        model: [ optiplex_9010_sff ]
```

Reply:
I agree, changed in [this commit](https://github.com/Dasharo/coreboot/pull/541/commits/0a7ec66884e093d282af3e56760359b4c0fc14bf)

Reply:
> I think we don't plan to release anything else other than UEFI and TXT enabled config, so for better readability of the binary artifact these changes should be better

I am afraid we will need the seabios version as well, at least for trenchboto testing, as suggested by @krystian-hebel , correct?

Reply:
Yes, unfortunately we need legacy to have a baseline, so either SeaBIOS or (unlikely) CSM will be required.

Reply:
Alright, looks like [this](https://github.com/Dasharo/coreboot/pull/541/commits/e7799db8df211bc305cd773af9f5e6cc63361017) does the trick

Reply:
Please confirm @macpijan , @krystian-hebel